### PR TITLE
Move parse_integer and parse_unsigned_integer from util::str to style::attr

### LINKED ISSUE #9637: Move parse_integer and parse_unsigned_integer from util::str to style::attr
The underlying function `do_parse_integer` should be moved too.

Files:
 * components/util/str.rs
 * components/style/attr.rs

### PR DESCRIPTION
**Pull request by zakorgy**
Fixes #9637

[Review on Reviewable](https://reviewable.io/reviews/servo/servo/9657)

### CODE REVIEW

**Comment by KiChjang** [comment]
@zakorgy Why did you close it?

**Comment by zakorgy** [comment]
Havent committed everything

**Comment by KiChjang** [comment]
I see 3 commits, you've already commited something and this is ready to be reviewed.

**Comment by KiChjang** [comment]
Just a tip - you don't need to close and reopen PRs in order to add commits to it, you can continuously push commits to your branch, and the GH PR interface will still be able to pick up all the changes you've made.

**Comment by zakorgy** [comment]
Thanks i will take your advice. I just panicked when i saw, that i still have uncomitted changes after rebase.

File: components/style/attr.rs [hunk 1]
```diff
@@ -5,12 +5,13 @@
...
 use cssparser::{self, Color, RGBA};
 use euclid::num::Zero;
+use num::ToPrimitive;
 use std::ascii::AsciiExt;
 use std::ops::Deref;
 use string_cache::{Atom, Namespace};
 use url::Url;
-use util::str::{DOMString, LengthOrPercentageOrAuto, WHITESPACE, parse_unsigned_integer, parse_length};
-use util::str::{split_html_space_chars, str_join, parse_integer};
+use util::str::{DOMString, HTML_SPACE_CHARACTERS, WHITESPACE, LengthOrPercentageOrAuto};
```
Comment (KiChjang) [inline components/style/attr.rs]: nit: Put the constants to the right, and bring `LengthOrPercentageOrAuto` right after `DOMString`.

**Comment by KiChjang** [comment]
This is fine to merge after my small nit.

**Comment by KiChjang** [comment]
Squash!

**Comment by KiChjang** [comment]
@bors-servo r+

Thanks for your contribution!

**Pull request merged**